### PR TITLE
Update EdgeCloud_LcM.yaml

### PR DESCRIPTION
**Pull request by gunjald**
Improving the description of the username field

### CODE REVIEW

File: code/API_definitions/EdgeCloud_LcM.yaml [hunk 1]
```diff
@@ -436,7 +436,7 @@ components:
               $ref: '#/components/schemas/Uri'
             userName:
               type: string
-              description:  Username to acces the artifact repository
+              description:  Username is mandatory to access the artifact repository in case of private repository
             password:
               type: string      
               description:  Password to acces the artifact repository
```
Comment (sergiofranciscoortiz) [inline code/API_definitions/EdgeCloud_LcM.yaml:442]: Correct spelling mistake: Password to access

Comment (sergiofranciscoortiz) [reply code/API_definitions/EdgeCloud_LcM.yaml:442]: Also for token description: Authorization token **to** acces**s**

**Review by sergiofranciscoortiz — CHANGES_REQUESTED** [review]
Change request makes sense. While reviewing also spotted a couple of spelling mistakes for password and token parameter in Uri schema. Please add the correction before merging.

File: code/API_definitions/EdgeCloud_LcM.yaml (hunk 1 above)
Comment (gunjald) [reply code/API_definitions/EdgeCloud_LcM.yaml]: Changes done and committed

**Review by sergiofranciscoortiz — APPROVED** [review]

**Review by gainsley — APPROVED** [review]

**Comment by sergiofranciscoortiz** [comment]
Merging as no further comments for a while.

**Pull request merged**